src/status_im/anon_metrics/transformers.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.anon-metrics.transformers
+  "`status-go` defines the shape of expected events so that we don't overcollect data (by mistake or intentionally).
+  `transformers` ns transform the event payload to match the expected shape on `status-go` side.")
+
+(defn navigate-to-txf [context]
+  {:view-id (second context)
+   :params (-> context
+               (nth 2)
+               (select-keys [:screen]))})
+
+(def transformations
+  {:navigate-to navigate-to-txf})
+
+(defn transform [ctx]
+  (when-let [txf (-> ctx first transformations)]
```

Comment:
could please explain this line `(-> ctx first transformations)` i can't understand it , thanks

Reply:
This is the approximate shape of the `ctx` map:
```clojure
 (def ctx
    [:navigate-to ; event-type
     :home ; view-id
     {:screen "screen-naame"
      :some :other
      :element :ts}])
```

We want to lookup a transformation against `event-type`, i.e. the first element of the vector:

* `(-> ctx first) ;; => :navigate_to`
* `(-> :navigate_to transformations) ;; => the transformation for :navigate-to event`

Reply:
but we call it here with re-frame context `transformed-payload (txf/transform context)` probably we should call with `(get-in context [:coeffects :event])`  then?

Reply:
yes you are right. This is a bug, the shape of `ctx` is not [the vector I commented above](https://github.com/status-im/status-react/pull/11901/files#r600299010). That vector is the shape of `(-> context :coeffects :event)`. 

Fixing it.